swarm/storage/common_test.go

```diff
@@ -184,13 +183,32 @@ func testStoreCorrect(m ChunkStore, processors int, n int, chunksize int64, t *t
 func benchmarkStorePut(store ChunkStore, processors int, n int, chunksize int64, b *testing.B) {
 	b.ReportAllocs()
 	b.ResetTimer()
```

Comment:
RestTimer probably needs to be after the new code, just before the for loop.

Reply:
Fixed in 6dce5c198f9b1de891d5713bed780e6c764690f0